tutorials/continuous-aggs-tutorial.md

```diff
@@ -0,0 +1,238 @@
+
+# Tutorial: Continuous Aggregates
+
+One kind of query that occurs frequently when performing data analysis is
+aggregating data into summaries.
+
+For instance if we want to find out the average of a column for each day in the 
+dataset stored in a hypertable, we would run 
+
+``` sql
+SELECT time_bucket('1 day', timedimesion-column-name), avg(column-name), stddev(column-name)
+FROM hypertable;
+```
+
+However, performing this query as written requires scanning all the data within
+the table, which can be inefficient if this query is called frequently.
+A continuous aggregate query computes the query results automatically at a user 
+specified time interval without the user having to repeatedly rerun the query. 
+This is particularly useful when recomputing aggregates frequently for large 
+data sets.
+
+We will explore this feature using the data set from the [Hello NYC Tutorial][hello_nyc]
+
+Prerequisites:
+TimescaleDB installed (version 1.3 or greater)
+
+### 1. Download and Load Data
+
+Let's start by downloading the dataset.
+
+This dataset contains two files:
+1. `contagg_nyc_data.sql` - A SQL file that will set up the necessary tables
+for the continuous aggregates tutorial.
+1. `nyc_data_rides.csv` - A CSV file with the ride data
+
+First, create a database, e.g., `nyc_data` with the extension:
+
+```sql
+CREATE DATABASE nyc_data;
+\c nyc_data
+CREATE EXTENSION IF NOT EXISTS TimescaleDB CASCADE;
```

Comment:
does 'timescaledb' need to be all lowercase here?

Reply:
fixed